### PR TITLE
config: enable tichi for tidb-tools

### PR DESCRIPTION
**Pull request by hi-rustin**
Part of https://github.com/ti-community-infra/configs/issues/173

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

- [ ] Migration SIG check config (@lance6716)
- [ ] @sykp241095 Give the bot admin permission

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -656,6 +677,32 @@ ti-community-blunderbuss:
...
       - tiancaiamao
+  - repos:
+      - pingcap/tidb-tools
+    pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
+    max_request_count: 2
+    exclude_reviewers:
+      # Bots
+      - ti-chi-bot
+      - ti-srebot
+      # Inactive reviewers for dumpling.
```
Comment (lance6716) [inline prow/config/external_plugins_config.yaml:688]:
```suggestion
      # Inactive reviewers for tidb-tools.
```

**Review by lance6716 — COMMENTED** [review]
prefer to remove 

amyangfei
glorv
GMHDBJD
leoppro
YuJuncens
overvenus
Little-Wallace
liuzix

**Comment by hi-rustin** [comment]
> prefer to remove
> 
> amyangfei
> glorv
> GMHDBJD
> leoppro
> YuJuncens
> overvenus
> Little-Wallace
> liuzix

Fixed.

**Comment by hi-rustin** [comment]
@lance6716  PTAL~

**Review by lance6716 — APPROVED** [review]

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ thanks!

**Review by Mini256 — APPROVED** [review]

**Pull request merged**